content/blog/cdcon+gitopscon-at-open-source-summit/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+
+On December 14th, the CNCF [announced changes to how co-located events will work](https://www.cncf.io/blog/2022/12/14/important-updates-for-2023-kubecon--cloudnativecon-and-co-located-events/) moving forward, starting with KubeCon+CloudNativeCon 2023 EU.
+
+With this announcement, we are happy to announce that GitOpsCon is joining forces with CdCon and moving from KubeCon + CloudNativeCon to Open Source Summit starting in 2023!
+
+This relocation to Open Source Summit - also a Linux Foundation event - is planned to continue moving forward, both in North America and Europe, and more directly align CD Foundation and CNCF communities. Everyone is welcome!
+
+## Upcoming dates
+
+| | |
+| -- | -- |
+| April 2023 | The GitOps WG and OpenGitOps will continue to have an onsite presence at [KubeCon + CloudNativeCon, EU 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/) in Amsterdam, NL - as a project meeting or ½ day co-located event - but this will no longer be where the main GitOpsCon event is held. In addition, CNCF GitOps projects such as Argo, Carvel, Flux, and Keptn will be in the project pavilion as usual. |
+| May 2023 | This coming GitOpsCon will be at [Open Source Summit, North America 2023](https://events.linuxfoundation.org/open-source-summit-north-america/) from May 10-12 in Vancouver, Canada. This will be a 2-day, multi-track event, in a larger space to accommodate even more attendees! A CNCF booth is also planned to be next to the CDFoundation booth, to represent CNCF groups including the GitOps WG, and projects relevant to the colocated events including GitOps CNCF projects above. |
+| September 2023 | The following GitOpsCon would be at [Open Source Summit, Europe 2023](https://events.linuxfoundation.org/open-source-summit-europe/) in Bilbao, Spain. |
+| November 2023 | Continuing with GitOps WG and OpenGitOps project meeting at [KubeCon + CloudNativeCon, NA 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2023/) in Chicago, IL. |
+
+## Context
+
+GitOpsCon will share this event with CdCon, and co-organize this with the CD Foundation. This is a huge alignment between CNCF and the CDF, and we are honoured that GitOpsCon can serve as the gateway for this collaboration
+While this is also part of a larger reorganisation of co-located events at KubeCon+CloudNativeCon,  GitOpsCon in particular has been the most well-attended KCCNC co-located event, so it is a big announcement that it is now big enough to move to this kind of joint event at Open Source Summit
+
+## Communication
+
+- On Dec 8 LF informed the GitOpsCon organisers of this change by email. We have not yet announced this to the rest of the WG in the week between then and now because there has not been a WG meeting, and we have been in discussion with LF this past week to gather details before presenting to the group. As of our meeting with Linus Foundation staff today, Dec 15, we now have more complete info to share.
```

Comment:
Little typo? Linus Foundation or Linux Foundation

Reply:
@niklasmtj Fixed in #109 with co-author credit to you 💖